akka-remote/src/main/resources/reference.conf

```diff
@@ -669,6 +669,10 @@ akka {
         # The first step ensures that all nodes will send over a certificate when asked to. The second
         # step will ensure that all nodes finally enforce the secure checking of client certificates.
         require-mutual-authentication = on
+
+        # The class given here must implement the akka.remote.transport.netty.CustomTrustManagerFactory
+        # interface and offer a public constructor with no arguments:
+        trust-manager-factory-class = "akka.remote.transport.netty.DefaultTrustManagerFactory"
```

Comment:
could this be related to https://github.com/akka/akka/pull/23670 ?
should we have a provider for all such security things that should be pluggable?